docs/hyn/5.2/Logging.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Logging
+icon: fal fa-book
+---
+
+Especially in production you might want to split the default laravel log by tenants.
+Tenancy shipps with a custom tenant aware logger which you can register in the logger config.
+
+If a valid hostname is detected it will create a `logs` folder within the tenant directory ie.:
+`app/tenancy/tenants/xxxxxxxxxxxxxxxxxxxx/logs/{log_level}_xxxx-xx-xx.log`
+
+If no valid hostname is detected it will default to:
+`/storage/logs/{log_level}_xxxx-xx-xx.log`
+
+Add a new `tenant` channel to the channels array in your **config/logger.php** like below:
+
+```php
+...
```

Comment:
I don't think the `...` are necessary.  Someone may think they are, copy/paste them, then complain it's not working.  One would need to be very novice to do so, but it's entirely plausible.

Reply:
Agreed.